src/constants.py

```diff
@@ -80,7 +80,7 @@
     "summer": "5/17/20-8/27/20",
 }
 SQLITE_MAX_VARIABLE_NUMBER = 999
-STATIC_SOURCES_URL = GIT_CONTENT_URL + "/eatery-backend/master/static_sources/"
+STATIC_SOURCES_URL = GIT_CONTENT_URL + "/eatery-backend/raahi/openTable-get/static_sources/"
```

Comment:
why does this include your name

Reply:
oh yikes I forgot to change the url back to master thanks